package.json

```diff
@@ -51,13 +51,13 @@
     "example:basic": "cd ./boilerplates/react-union-boilerplate-basic && yarn start",
     "lint": "eslint \"packages/**/*.js\" \"boilerplates/**/*.js\" --cache",
     "lint:fix": "yarn lint --fix",
-    "pretty": "prettier --write ./packages/**/*.js ./boilerplates/**/*.js && yarn lint:fix",
+    "pretty": "prettier --write",
```

Comment:
Do you insist on keeping the script there? I personally never use Prettier alone, only with ESLint, so I'd remove this line completely.